app.json

```diff
@@ -36,17 +36,19 @@
             "required": false
         },
         "HMD_ALLOW_ORIGIN": {
```

Comment:
So here actually a mistaken from @Yukaii 
It should be `HMD_PORT` instead of `HMD_ALLOW_ORIGIN `.